challenge2-incremental/incremental_client.py

```diff
@@ -73,7 +72,7 @@ def main():
         synthetic_local_text_data = model.submodel_zero(training_data)
 
         # Load Weights and Training Data and Process it through submodel One
-        updated_weights = model.submodel_one(global_parameters_path, training_data)
+        updated_weights = model.submodel_one(initial_weights, training_data)
```

Comment:
I have replaced the file path with the initial weights taken from the server. As the client will not have a path to the the pt file

Reply:
This makes sense. 
And we will have to define the initial_weights = get_initial_server_data() before this.